util/ssh_client.go

```diff
@@ -29,6 +51,112 @@ func (c *sshClientImpl) ExecCommand(username string, password string, ip string,
 	return string(output), err
 }
 
+func (c *sshClientImpl) UploadFile(username string, password string, ip string, srcFile string, destFile string) error {
```

Comment:
Here they should be public as you did. You can note also that these are implementing the interface type. Thx.